src/utils/interface.js

```diff
@@ -1,3 +1,6 @@
+
+const KEY_CODES = { q: 81, v: 86 };
```

Comment:
Actually that's a good question, can you investigate? I would assume that keycodes are just the physical keys that are pressed, and so doesn't matter if you pressed shift or not